### PR TITLE
OPSEXP-2460: update README to reflect new release policy

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2460

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,73 +49,50 @@ account.
...
+the product versions. We will also stick to Semver principles when choosing
+next version number, meaning that:
+
+* patch version will be used for bug fixes (last digit)
+* minor version will be used for new features and modifications which do not
+  introduce breaking changes in the configuration interface.
+* major version will be used for changes which involve breaking changes in the
+  configuration interface.
+
+The `alfresco-conetnt-services` chart has always provided the ability to deploy
```
Comment (gionn) [inline README.md:69]:
```suggestion
The `alfresco-content-services` chart has always provided the ability to deploy
```

File: README.md [hunk 2]
```diff
@@ -49,73 +49,50 @@ account.
...
+deploy (by default). Instead we'll just maintain the list of deprecated versions
+versions mapped with the latest versions of the charts we tested deployment
+with, so you can use that version to deploy older ACS version on Kubernetes.
+Check the [ACS End of Life'd versions](#acs-end-of-lifed-versions) paragraph.
+
+You are encouraged to always use the latest version of the chart to deploy your
+currently supported ACS version, using the appropriate values file.
+
+Finally, the master branch of this repository used to contain the latest
+versions, including non-released, versions!
```
Comment (gionn) [inline README.md:83]:
```suggestion
versions, including non-released versions!
```

File: README.md [hunk 3]
```diff
@@ -49,73 +49,50 @@ account.
...
+We're also moving away from this pattern and the chart will now only ever
+deploy released versions of our products.
+
+Should you want to try our latest dev versions, we now provide an additional
+values file called `pre-realeses_values.yaml` which will be bumped on a regular
```
Comment (gionn) [inline README.md:88]: check file

File: README.md [hunk 4]
```diff
@@ -49,73 +49,50 @@ account.
...
-| [6.0.1][6.0.1]         | 7.4.0.1    |         | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [6.0.2][6.0.2]         | 7.4.0.1    |         | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [6.1.0-M.1][6.1.0-M.1] | 23.1.0-A19 | 7.4.0.1 | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [7.0.0-M.1][7.0.0-M.1] | 23.1.0-A27 | 7.4.1   | 7.3.1   | 7.2.1.12 | 7.1.1.8  | 7.0.1.10 | 7.4.1     |
-| [7.0.0-M.2][7.0.0-M.2] | 23.1.0-M4  | 7.4.1.1 | 7.3.1   | 7.2.1.12 | 7.1.1.8  | 7.0.1.10 | 7.4.1.1   |
-| [7.0.0][7.0.0]         | 23.1.0     | 7.4.1.2 | 7.3.1.1 | 7.2.1.12 | 7.1.1.8  | 7.0.1.10 | 23.1.0    |
-| [7.0.1][7.0.1]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
-| [7.0.2][7.0.2]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
-| [7.0.3][7.0.3]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
-| [8.0.0-M.1][8.0.0-M.1] | 23.2.0-M1  | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
```
Comment (gionn) [inline README.md:92]: maybe we should keep it for reference? maybe in a subpage

Comment (alxgomz) [reply README.md:92]: I don't think it has a lot of value and is probably more keeping people away from using the latest chart should they want to deploy an earlier version of the platform.

File: README.md [hunk 5]
```diff
@@ -49,73 +49,50 @@ account.
...
+We're also moving away from this pattern and the chart will now only ever
+deploy released versions of our products.
+
+Should you want to try our latest dev versions, we now provide an additional
+values file called `pre-release_values.yaml` which will be bumped on a regular
+basis.
+
+This also means we will not produce `-M*` versions of the chart anymore.
+
+Check the [Release page](./releases) for the list of existing versions.
```
Comment (gionn) [inline README.md:93]:
```suggestion
Check the [Release page](https://github.com/Alfresco/acs-deployment/releases) for the list of existing versions.
```

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**